sdk/agrifood/arm-agrifood/src/azureAgFoodPlatformRPService.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreClient from "@azure/core-client";
+import * as coreRestPipeline from "@azure/core-rest-pipeline";
+import {
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest
+} from "@azure/core-rest-pipeline";
+import * as coreAuth from "@azure/core-auth";
+import {
+  ExtensionsImpl,
+  FarmBeatsExtensionsImpl,
+  FarmBeatsModelsImpl,
+  LocationsImpl,
+  OperationsImpl,
+  PrivateEndpointConnectionsImpl,
+  PrivateLinkResourcesImpl
+} from "./operations";
+import {
+  Extensions,
+  FarmBeatsExtensions,
+  FarmBeatsModels,
+  Locations,
+  Operations,
+  PrivateEndpointConnections,
+  PrivateLinkResources
+} from "./operationsInterfaces";
+import { AzureAgFoodPlatformRPServiceOptionalParams } from "./models";
+
+export class AzureAgFoodPlatformRPService extends coreClient.ServiceClient {
```

Comment:
Could you help add below into [typescript.readme](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/agrifood/resource-manager/readme.typescript.md) then re-generate this pr?

```
title: AgriFoodMgmtClient
````